Geometry/MTDGeometryBuilder/interface/RectangularMTDTopology.h

```diff
@@ -136,7 +136,7 @@ class RectangularMTDTopology final : public PixelTopology {
   bool isItEdgePixelInY(int iybin) const override { return ((iybin == 0) | (iybin == (m_ncols - 1))); }
```

Comment:
It isn't obvious that `||` would be faster. The "short-circuit" evaluation means that the `||` operator must come with a branch, unlike `|`. However, looking at this particular method Clang generates exactly the same code (with optimisation enabled - at least -O1):
<img width="1256" alt="Screenshot 2023-01-15 at 16 18 43" src="https://user-images.githubusercontent.com/1390682/212549895-39698a96-b69c-46f0-8a59-fbd39dd30bde.png">